servers/rendering_server.h

```diff
@@ -466,6 +466,7 @@ class RenderingServer : public Object {
 
 	virtual void multimesh_set_mesh(RID p_multimesh, RID p_mesh) = 0;
 	virtual void multimesh_instance_set_transform(RID p_multimesh, int p_index, const Transform3D &p_transform) = 0;
+	virtual void multimesh_instance_set_transform_teleport(RID p_multimesh, int p_index, const Transform3D &p_transform) = 0;
```

Comment:
This should probably more sensibly hook in with the existing `multimesh_instance_reset_physics_interpolation()` on line 494. The current transform should already be available in the server if required. When physics interpolation is in use, the (interpolated) transform won't be available anyway scene side.

_(`Multimeshes` are not changed in the new `SceneTreeFTI` BTW as there are no pressing reasons to move the multimesh instances scene side.)_

The naming isn't ideal currently, but we don't want to be calling two separate server functions here to do the same thing. If necessary we can consider renaming in a followup but it would be compat breaking.